src/MainPage/MainPage.js

```diff
@@ -42,13 +61,66 @@ function MainPage() {
 
       <div className='play-container'>
         <div className={`play-area-${Math.round(Math.random() * bgArray.length)}`} >
-          <button type="button" className='help-button'>
-            Help
-          </button>
-        </div>
-      </div>
+          <div className='button-containers-1'>
+            <button type="button" className='help-button' onClick={handleHelpVisible}>
+              ?
+              <Modal style={{ display: 'block', position: 'center' }}
 
+                show={showHelp} onHide={handleHelpVisible}>
+                <Modal.Header closeButton>
+                  <Modal.Title>Help</Modal.Title>
 
+                </Modal.Header>
+                <Modal.Body>
+                  Help page will go here
+                  {/* <PartyMenu pokemon1={"pokemon1"} pokemon2={"pokemon2"} pokemon3={"pokemon3"} /> */}
+                </Modal.Body>
+                <Modal.Footer>
+                  <Button variant="secondary" onClick={handleHelpVisible}>
+                    Close
+                  </Button>
+                </Modal.Footer>
+              </Modal>
+            </button>
+          </div>
+          <div className='button-containers-2'>
+
+            <button type="button" className='train-button'>
```

Comment:
Excellent structure, I appreciate the attention to detail with semantic HTML for keeping project accessible.